common/locales/en/defaultTasks.json

```diff
@@ -1,21 +1,19 @@
 {
-  "defaultHabit1Text": "Productive Work (Click the pencil to edit)",
```

Comment:
@Alys So does that mean that where we might find json files that do not have 2 spaces per indent, we should convert them to the 2-space standard, or just leave all indentation as is? From what I saw of the locales other than _en_, they all were 4-spaced and had no empty lines like _en/defaultTasks.json_.